actions/image_partition_action.go

```diff
@@ -498,7 +511,11 @@ func (i ImagePartitionAction) Run(context *debos.DebosContext) error {
 
 		command := []string{"parted", "-a", "none", "-s", "--", context.Image, "mkpart", name}
 		switch p.FS {
-		case "vfat":
+		case "fat16":
+			command = append(command, "fat16")
+		case "fat", "fat12", "fat32", "msdos", "vfat":
+			/* TODO: Not sure if this is correct. Perhaps
+			   fat12 should be treated the same as fat16 ? */
```

Comment:
fat12 has it's own unique partition id on msdos labels ; but parted seems to only recognize this, you can't create one. So :man_shrugging: